modules/h2/src/main/java/org/h2/command/dml/Call.java

```diff
@@ -66,7 +65,7 @@ public ResultInterface query(int maxrows) {
         if (isResultSet) {
             return v.getResult();
         }
-        LocalResult result = LocalResultFactory.DEFAULT.create(session, expressions, 1);
+        LocalResult result = session.getDatabase().getResultFactory().create(session, expressions, 1, true);
```

Comment:
@tledkov-gridgain, please re-check this.
Looks like this cause unneeded MemoryTracker creation and memory allocation.